contributors.yml

```
@@ -70,3 +70,5 @@
 - AxeemHaider
 - alexvdvalk
 - thame
+- christianrr
+- cr-mutor
```

Let's only add the current username, we cannot 100% verify the other one
```suggestion
- christianrr
```

---

Thanks! 🚀